src/content/__tests__/monetizationLinkManager.test.ts

```diff
@@ -0,0 +1,1236 @@
+import { JSDOM } from 'jsdom';
+import { MonetizationLinkManager } from '@/content/services/monetizationLinkManager';
+import type {
+  ContentToBackgroundMessage,
+  MessageManager,
+  Response,
+} from '@/shared/messages';
+import type { Logger } from '@/shared/logger';
+import type { WalletAddress } from '@interledger/open-payments';
+import { success, failure } from '@/shared/helpers';
+
+// for syntax highlighting
+const html = String.raw;
+
+const WALLET_ADDRESS: WalletAddress[] = [
```

Comment:
Can rename this to `WALLET_INFO` and create `WALLET_ADDRESS = WALLET_INFO.map(e => e.id);` then we don't have to use `WALLET_ADDRESS[0].id` everything in tests, and can do `WALLET_ADDRESS[0]` directly.